.github/scripts/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "okp4-readme",
+  "version": "1.0.0",
+  "description": "test",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/alexandrglaznyov/okp4-README.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/alexandrglaznyov/okp4-README/issues"
+  },
+  "homepage": "https://github.com/alexandrglaznyov/okp4-README#readme",
```

Shouldn't we put the correct repo url here? @ccamel  

```suggestion
    "url": "git+https://github.com/okp4/awesome.git"
  },
  "author": "",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/okp4/awesome/issues"
  },
  "homepage": "https://github.com/okp4/awesome#readme",
```